### ISSUE TITLE
Installed API always only returns empty arrays

### ISSUE THREAD

**Issue by KathleenLD** [mapzen/vector-datasource#128]
*We got a support request which I'm moving here, per @rmarianski's suggestion*

> We followed your installation guide on https://github.com/mapzen/vector-datasource/wiki/Mapzen-Vector-Tile-Service# https://github.com/mapzen/vector-datasource/wiki/Mapzen-Vector-Tile-Service# and found that the installed API always only returns empty arrays, even though, we have downloaded the latest Switzerland file from geofabrik.
>
> There were also no errors that occurred during the installation process.

**Comment by rmarianski** [comment]
What happens if you try a url like: /all/1/1/1.json ? Data should exist there, regardless of the extract (comes from the natural earth dataset)

**Comment by salam** [comment]
Thanks @KathleenLD. 

@rmarianski, it actually returns the natural earth dataset, but not the Swiss pack from geofabrik, as it seems:

Tile /all/5/16/11.json:
![Tile /all/5/16/11.json](https://cloud.githubusercontent.com/assets/432008/9248147/b125c00e-41b9-11e5-9445-308e4aa6c2da.png)

Maybe the following command does not fill it in correctly?
```bash
osm2pgsql --hstore --create -C 750 -U postgres -S vector-datasource/osm2pgsql.style -d osm -k switzerland-latest.osm.pbf -l -H localhost -W
```

Or did screw up the user permissions with postgres/osm? But then, even the natural earth dataset would not be shown, right?

Thanks a lot!

Matthias

**Comment by rmarianski** [comment]
@salam 

I tried downloading the latest Switzerland file, and when I loaded it locally it did seem to contain data.

I just noticed that your osm2pgsql command line specified the latlong flag. The data is expected to be stored in mercator coordinates. If you try the data load process again but leave that flag off, it should work.

I see that the documentation had the latlong flag specified too. I just corrected it. Sorry about that!
